config/environments/production.rb

```diff
@@ -16,13 +16,12 @@
   config.consider_all_requests_local       = false
   config.action_controller.perform_caching = true
 
-  # Ensures that a master key has been made available in either ENV["RAILS_MASTER_KEY"]
-  # or in config/master.key. This key is used to decrypt credentials (and other encrypted files).
+  # Ensures that a master key has been made available in ENV["RAILS_MASTER_KEY"], config/master.key, or an environment
+  # key such as config/credentials/production.key. This key is used to decrypt credentials (and other encrypted files).
   # config.require_master_key = true
 
-  # Disable serving static files from the `/public` folder by default since
-  # Apache or NGINX already handles this.
-  config.public_file_server.enabled = ENV["RAILS_SERVE_STATIC_FILES"].present?
+  # Enable static file serving from the `/public` folder (turn off if using NGINX/Apache for it).
+  config.public_file_server.enabled = true
```

Comment:
Are we not using `RAILS_SERVE_STATIC_FILES` env var in forms-deploy? if not  then we should remove it

Reply:
Ah damn I thought we should keep it but in fact we should remove from forms-deploy... I'll add that to the other PR and bring it back here...

Reply:
On second thought it's an incoming change and we don't already have this set in forms-deploy so leaving it as the default. We are currently serving a commented out robots.txt but I do think we should actually set something there...